core/src/main/java/org/apache/accumulo/core/data/Mutation.java

```diff
@@ -962,6 +973,25 @@ public QualifierOptions family(Text colFam) {
       return family(colFam.getBytes(), colFam.getLength());
     }
 
+    /**
+     * Sets the column family, column qualifier, and column visibility of a mutation. All other
```

Comment:
Could remove this javadoc.  I think alot of tooling will show the parent javadoc when nothing is present, so do not need to maintain two copies of the docs.